packages/cli/src/commands/labs/index.ts

```diff
@@ -0,0 +1,87 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import {Args, ux} from '@oclif/core'
+import {sortBy} from 'lodash'
+
+interface Features {
+  currentUser: any,
+  user: any,
+  app?: any,
+}
+
+function printJSON(features: Heroku.Account | Heroku.AccountFeature | Heroku.AppFeature) {
+  ux.log(JSON.stringify(features, null, 2))
+}
+
+function printFeatures(features: any) {
+  features = sortBy(features, 'name')
+  const longest = Math.max.apply(null, features.map((f: Record<string, string>) => f.name.length))
+  for (const f of features) {
+    let line = `${f.enabled ? '[+]' : '[ ]'} ${f.name!.padEnd(longest)}`
```

Comment:
I understand this is likely a carry over from legacy code but in general, the use of these non-null assertions using `!` should be avoided. They are the TypeScript version of "hold my beer". 

Expressive types won't necessitate their use and in the case of objects with optional properties inside a template literal, an optional chaining operator combined with a nullish coalescing operator will suffice:

```suggestion
    let line = `${f.enabled ? '[+]' : '[ ]'} ${f.name?.padEnd(longest) ?? '' }`
```